pkg/minishift/addon/addon_meta.go

```diff
@@ -31,19 +31,28 @@ const (
 // AddOnMeta defines a set of meta data for an AddOn. Name and Description are required. Others are optional.
 type AddOnMeta interface {
 	Name() string
-	Description() string
+	Description() []string
 	RequiredVars() []string
 	GetValue(key string) string
 }
 
 type DefaultAddOnMeta struct {
-	headers map[string]string
+	headers map[string]interface{}
 }
 
-func NewAddOnMeta(headers map[string]string) (AddOnMeta, error) {
+func NewAddOnMeta(headers map[string]interface{}) (AddOnMeta, error) {
 	for _, tag := range requiredMetaTags {
-		if headers[tag] == "" {
-			return nil, errors.New(fmt.Sprintf("Metadata does not contain an mandatory entry for '%s'", tag))
+		switch v := headers[tag].(type) {
+		case string:
+			if v == "" {
+				return nil, errors.New(fmt.Sprintf("Metadata does not contain an mandatory entry for '%s'", tag))
+			}
+		case []string:
+			if len(v) == 0 {
+				return nil, errors.New(fmt.Sprintf("Metadata does not contain an mandatory entry for '%s'", tag))
+			}
+		default:
+			return nil, nil
```

Comment:
@hferentschik Does that what you wanted as part of having multiline string for a specific key?